sap/adt/core.py

```diff
@@ -216,30 +374,38 @@ def _get_session(self):
             self._session = requests.Session()
             self._session.auth = self._auth
             # requests.session.verify is either boolean or path to CA to use!
-            self._session.verify = os.environ.get('SAP_SSL_SERVER_CERT', self._session.verify)
+            self._session.verify = os.environ.get('SAP_SSL_SERVER_CERT',
+                                                  self._session.verify)
 
             if self._session.verify is not True:
-                mod_log().info('Using custom SSL Server cert path: SAP_SSL_SERVER_CERT = %s', self._session.verify)
+                mod_log().info(
+                    'Using custom SSL Server cert path: SAP_SSL_SERVER_CERT = %s',
+                    self._session.verify)
             elif self._ssl_verify is False:
-                import urllib3
```

Comment:
How is this change related to ADT over RFC?

Reply:
Not related. I reverted it.